lib/solitude/api.py

```diff
@@ -851,10 +849,26 @@ def create_transaction(self, generic_buyer,
             'application_size': application_size,
             'source': source
         })
+
         bill_id = res['billingConfigurationId']
-        log.info('transaction {tr}: billing config ID: {bill}; '
-                 'prices: {pr}'
-                 .format(tr=transaction_uuid, bill=bill_id, pr=prices))
+        log.info('{pr}: made trans billing config ID: {bill}; '
+                 'transaction {tr}; prices: {prices}'
+                 .format(tr=transaction_uuid, bill=bill_id, prices=prices,
+                         pr=self.name))
+
+        trans = self.slumber.generic.transaction.post({
+            'provider': solitude_const.PROVIDERS[self.name],
+            'buyer': generic_buyer['resource_uri'],
+            'seller': generic_seller['resource_uri'],
+            'seller_product': generic_product['resource_uri'],
```

Comment:
I wonder if this needs to be ``provider_product`` but I admit I'm confused now when trying to figure it out